erpnext_documentation/www/docs/user/manual/en/accounts/dunning.md

```diff
@@ -0,0 +1,50 @@
+<!-- add-breadcrumbs -->
+# Dunning
+
+**A document to be sent as a persistent demand for debt payment.**
+
+Dunning is a document to store and send as a persistent demand for debt payment against an unpaid Sales Invoice.
+
+To access the Dunning list, go to:
+> Home > Accounting > Dunning
+
+## 1. Prerequisites
+Before creating a Dunning list, there must be a Sales Invoice since it is created against it.
+
+* [Sales Invoice](/docs/user/manual/en/accounts/sales-invoice)
+* [Customer](/docs/user/manual/en/CRM/customer)
```

Comment:
```suggestion
```

Customer is already a prerequisite for Sales Invoice, so it's kind of redundant here.